app/App.js

```diff
@@ -86,4 +86,4 @@ class App extends Component<{
   }
 }
 
-export default hot(module)(App);
+export default hot(App);
```

Comment:
Note: this change is unrelated to the fix. I'm just using the opportunity to move to the new API because `hot(module)(App)` is considered deprecated now (they instead want you to `import` from `/root` like I do here)